AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -262,6 +264,23 @@ - (void)signOut {
 
 #pragma mark - Private methods
 
+- (void)enableMSALLogging {
+  MSALGlobalConfig.loggerConfig.logLevel = MSALLogLevelVerbose;
```

Comment:
Would it make sense to match the MSAL log level to the App Center log level?

Reply:
Then it would be a bigger work. We need to add a delegate to `MSAppCenter` to listen to it whenever the log level changed. Log level can be changed anytime.

Reply:
I don't know how noisy MSAL is but it might be worth it.  If someone sets their AC log level very low and then MSAL starts spitting out tons of logs when they just wanted some basic ones, then it would be a bad experience

Reply:
Also, for servicing and debugging, it would mean the customer has to do 2 different things to get the most detailed logs, but if we did create the delegate then the troubleshooting steps remain the same for everyone

Reply:
We've talked about this one offline. Should be good, right?